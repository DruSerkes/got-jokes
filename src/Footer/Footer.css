.Footer {
	position: relative;
    width: 100%;
    margin-top: auto;
    margin-bottom: 0%;
    bottom: 0px;
    text-align: center;
	color: white;
	font-size: 1rem;
	border-top: 1px solid white;
}

.Footer p {
	padding-bottom: 0.3rem;
}

.Footer a {
	color: orange;
	text-decoration: none;
}