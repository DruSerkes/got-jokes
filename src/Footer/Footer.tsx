import * as React from 'react';
import './Footer.css';

export const Footer = () => {
  return (
    <footer className="Footer">
      <p>
        <a data-testid="footer" href="https://github.com/druserkes" target="_blank" rel="noopener noreferrer">
          Dru Serkes &copy;2021
        </a>
      </p>
    </footer>
  );
};