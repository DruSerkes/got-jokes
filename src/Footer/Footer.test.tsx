import * as React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer tests', () => {

    it('should render and match snapshot', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should render my name &copy2021', () => {
        const { getByText } = render(<Footer />);
        const myName = getByText(`Dru Serkes Â©2021`);
        expect(myName).toBeInTheDocument();
    });

}); 