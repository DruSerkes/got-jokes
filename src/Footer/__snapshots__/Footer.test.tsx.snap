// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests should render and match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="Footer"
  >
    <p>
      <a
        data-testid="footer"
        href="https://github.com/druserkes"
        rel="noopener noreferrer"
        target="_blank"
      >
        Dru Serkes Â©2021
      </a>
    </p>
  </footer>
</DocumentFragment>
`;
